src/inspector_agent.cc

```diff
@@ -477,6 +477,19 @@ class NodeInspectorClient : public V8InspectorClient {
     }
   }
 
+  inline bool waiting_for_frontend() { return waiting_for_frontend_; }
+
+  void StopWaitingForFrontend() {
+    waiting_for_frontend_ = false;
```

Comment:
If waiting for frontend is false, we can return early

Reply:
Done.